example-monorepos/with-tailwindcss/apps/expo/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "dependencies": {
+    "app": "*",
```

Comment:
this is missing `components`. let's rename `components` to `design`, and put it inside the `app` folder. I'm not sure if there's a reason to separate it from `app`, since it should work in all apps